LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-file.optional-expected.txt

```diff
@@ -0,0 +1,9 @@
+
+
+PASS file: Empty value
+PASS file: Auto width with a short filename
+FAIL file: Auto width with a long filename assert_greater_than: expected a number greater than 240 but got 121
```

Comment:
This failure is due to the fact that Wk1 on mac sets the displayed file name to "Data" rather than the actual file name.